puppet/modules/slave/manifests/postgresql.pp

```diff
@@ -16,6 +16,15 @@
           ensure => 'present',
           before => Class['postgresql::globals'],
         }
+        yumrepo { 'foreman-infra-el7':
```

Comment:
Looks like if you don't add a name, yum becomes verbose and now we get hourly cron mails with:
```
/etc/cron.hourly/0yum-hourly.cron:

Repository 'foreman-infra-el7' is missing name in configuration, using id
```

Mind adding one?

Reply:
Sure I'll make another PR

Reply:
https://github.com/theforeman/foreman-infra/pull/1305